spec/classes/foreman_proxy__config__spec.rb

```diff
@@ -764,4 +751,23 @@
       ])
     end
   end
+
+  context 'with puppet use_cache enabled' do
+    let :pre_condition do
+      'class {"foreman_proxy":
+        puppet_use_cache => true,
+      }'
+    end
+
+    it 'should create the cache_location' do
+      contain_file('/var/cache/foreman_proxy').with_ensure('directory')
```

Comment:
I really don't get why the tests are green, I still think there should be a dash and no underscore :)